src/Bundle/Resources/config/services/state.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Sylius\Resource\Doctrine\Common\State\PersistProcessor;
+use Sylius\Resource\State\Factory;
+use Sylius\Resource\State\FactoryInterface;
+use Sylius\Resource\State\Provider;
+use Sylius\Resource\State\ProviderInterface;
+use Sylius\Resource\State\Responder;
+use Sylius\Resource\State\ResponderInterface;
+use Sylius\Resource\Symfony\Request\State\ApiResponder;
+use Sylius\Resource\Symfony\Request\State\TwigResponder;
+use Sylius\Resource\Symfony\Response\ApiHeadersInitiator;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+return static function (ContainerConfigurator $container) {
+    $services = $container->services();
+    $parameters = $container->parameters();
+    $container->import('state/processor.php');
+    $container->import('state/provider.php');
+    $container->import('state/**/*.php');
```

Comment:
```suggestion
    $container->import('state/');
```
IIRC, should import the whole directory

Reply:
done

Reply:
loading the directory is not supported so, I just keep this following one
` $container->import('state/**/*.php');`